recipe/meta.yaml

```
@@ -31,7 +31,7 @@ requirements:
     - pkg-config
   host:
     - python
-    - cython 0.29
+    - cython 3.0.0
     - extension-helpers
```

```suggestion
    - extension-helpers >=1,<2
```
https://github.com/astropy/astropy/blob/v6.1.0/pyproject.toml#L130